app/views/shared/_comment.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_with url: user_post_comments_path(@user, @post), model: Comment do |form| %>
+  <%= form.hidden_field :post_id, value: @post.id %>
+  <%= form.text_area :text, placeholder: 'Text' %>
+  <div class='comment-button'>
+    <%= form.submit "Comment post" %>
+  </div>
+<% end %>
```

Comment:
Well done!!! 💯 
The forms for creating posts, comments, and likes work as expected. Congratulations!!! 🎆👏🏾 

![image](https://user-images.githubusercontent.com/38088818/215171837-ed9e4081-bb5b-45c1-8721-d6533a692520.png)